src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/ProPlanControls/ProPlanControls.spec.jsx

```diff
@@ -0,0 +1,915 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
```

Comment:
Looks like a big test, but it's really 3 tests of the same:
- for the basic plan
- for the pro monthly
- for the pro yearly
- for trialing